### PR TITLE
Ap 4279 add organisation seed data

### PR DESCRIPTION
**Pull request by kmahern**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-4279)

Created organisation_types database table and model.
Seeded data.

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

**Review by colinbruce — APPROVED** [review]
LGTM :+1:

**Review by naseberry — APPROVED** [review]

File: spec/populators/organisation_types_populator_spec.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+RSpec.describe OrganisationTypesPopulator do
+  it { expect(described_class).to respond_to(:call) }
+  it { expect(described_class.new).to respond_to(:call) }
+
+  describe ".call" do
+    subject(:call) { described_class.call }
+
+    before do
+      # clear out existing Organisation Type data - because it is transactional, the seeded data will be restored
+      OrganisationType.all.map(&:destroy!)
```
Comment (jsugarman) [inline spec/populators/organisation_types_populator_spec.rb:12]: Any reason not to do this instead?
```suggestion
      OrganisationType.destroy_all
```

File: spec/populators/organisation_types_populator_spec.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+RSpec.describe OrganisationTypesPopulator do
+  it { expect(described_class).to respond_to(:call) }
+  it { expect(described_class.new).to respond_to(:call) }
+
+  describe ".call" do
+    subject(:call) { described_class.call }
+
+    before do
+      # clear out existing Organisation Type data - because it is transactional, the seeded data will be restored
```
Comment (jsugarman) [inline spec/populators/organisation_types_populator_spec.rb:11]: do we need this comment in a test?

Comment (kmahern) [reply spec/populators/organisation_types_populator_spec.rb]: removed

**Review by jsugarman — APPROVED** [review]
👍 just a couple of minor comments on the spec

File: spec/populators/organisation_types_populator_spec.rb (hunk 1 above)
Comment (kmahern) [reply spec/populators/organisation_types_populator_spec.rb]: I don't think so, just followed the same pattern as the other populator specs 🙂

Comment (kmahern) [reply spec/populators/organisation_types_populator_spec.rb]: have changed 👍

**Pull request merged**